src/Controls/src/Core/Handlers/Shell/ShellItemHandler.Windows.cs

```diff
@@ -45,6 +45,12 @@ void OnNavigationViewLoaded(object sender, RoutedEventArgs e)
 			if (PlatformView is not null)
 				PlatformView.Loaded -= OnNavigationViewLoaded;
 
+			// Now that the NavigationView is fully loaded assign the MenuItemsSource if it
+			// hasn't been set yet. This avoids a crash observed when assigning during
+			// ConnectHandler on Native AOT builds.
+			if (PlatformView is NavigationView nv && nv.MenuItemsSource == null)
```

Comment:
The device tests "Shell TabBar Foreground Initializes Correctly" are failing on Windows. Could you check if is related with this changes?